docs/lib/auth/fragments/ios/signin_next_steps/70_done.md

```diff
@@ -0,0 +1 @@
+Signin flow is complete when you get `done` and the user is successfully authenticated into the app.
```

Comment:
```suggestion
Signin flow is complete when you get `done`. This means the user is successfully authenticated. As a convenience, the SignInResult also provides the `isSignedIn` property, which will be true if the next step is `done`.
```

Please confirm that `isSignedIn` is correct

Reply:
Added